release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -0,0 +1,111 @@
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | **(Preview)** Add support for .NET 6.0 Service Fabric applications | **Brief desc**: Service Fabric version 8.1 introduces programming model support for .NET 6.0 Service Fabric applications.<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add API support for updating application descriptions | **Brief desc**: Service Fabric version 8.1 introduces REST API support for updating Service Fabric applications. Customers can specify an application name and denote application parameters to be updated. These parameters include the maximum nodes, minimum nodes, application metrics and whether application capacities should be removed.<br>Corresponding PowerShell API is documented here: https://docs.microsoft.com/en-us/powershell/module/servicefabric/update-servicefabricapplication?view=azureservicefabricps.<br>**Usage**: Added UpdateApplication REST API<br>**Impact**: Customer is able to update application with REST API.<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add periodic ping between RA and RAP to detect IPC failure and stuck process | **Brief desc**: Add periodic ping between Reconfiguration Agent (RA) and Reconfiguration Agent Proxy (RAP) to detect IPC failure and stuck process<br>**Usage**: On every node, the Service Fabric agent on the node will periodically exchange a heartbeat message with the customer process, to ensure the IPC is working and the process is not frozen. When such issues have been detected (not a single heartbeat message has been successfully exchanged in 15 min by default), the process will be terminated to mitigate the issue.<br>**Impact**: Add periodic ping between RA and RAP to detect IPC failure and stuck process<br> |
+| **Windows - 8.1.316.9590<br>Ubuntu 16 - 8.1.320.1<br>Ubuntu 18 - 8.1.320.1** | **Feature** | Add support probes for non containerized applications | **Brief desc**: Support for liveness and readiness probe for non containerized applications<br> |
```

Comment:
Similar to above summary, I'd just make the description the current resolution text and if we can link out to docs on how to implement? Currently I only see this page https://docs.microsoft.com/en-us/azure/service-fabric/probes-codepackage which is under containers. Could generalize it more and move it up in the TOC.

Desc: "Added support for liveness and readiness probes for non containerized applications"

Reply:
done with the desc change.
As per the doc changes, the engg team is working on it.